extensions/wikia/DesignSystem/i18n/de.json

```diff
@@ -72,5 +72,22 @@
   "global-navigation-wikis-community-central": "Community Deutschland",
   "global-navigation-wikis-explore": "Wikis erforschen",
   "global-navigation-wikis-fandom-university": "Fandom Universität",
-  "global-navigation-wikis-header": "Wikis"
+  "global-navigation-wikis-header": "Wikis",
+  "notifications-no-notifications-message": "Noch keine Benachrichtigungen.",
+  "notifications-mark-all-as-read": "Alle als gelesen markieren",
+  "notifications-replied-by-multiple-users-with-title": "__mostRecentUser__ und __number__ weitere Benutzer haben auf __postTitle__ <b>geantwortet</b>",
```

Comment:
You need to update https://github.com/Wikia/app/blob/dev/extensions/wikia/DesignSystem/i18n/build.js#L6-L19 as this format won't work in MediaWiki. It should be:
```
$1 und $2 weitere Benutzer haben auf $3 <b>geantwortet</b>
```
See https://wikia-inc.atlassian.net/wiki/display/DESYS/RFC%3A+Storing+translations for the background.